### PR TITLE
File name will now be name of the question.

### PR DESCRIPTION
**Pull request by Kamushy**
Wanted to do this because it makes it easier if producing mutliple videos. Please let me know of any errors

### CODE REVIEW

**Comment by JasonLovesDoggo** [comment]
LGTM!

**Comment by hitesh3132** [comment]
it does work

**Comment by Proton45** [comment]
What would happen if the program chooses the same thread a second time? Would it just overwrite the file or would it break? Either way, it would be very good if the programm would just skip a thread that it has already been turned into a video. I thought of this, but I'm not sure how to implement it

**Comment by Kamushy** [comment]
> > What would happen if the program chooses the same thread a second time? Would it just overwrite the file or would it break? Either way, it would be very good if the program would just skip a thread that it has already been turned into a video. I thought of this, but I'm not sure how to implement it
> 
> already implemented in my fork
> 
> see [this commit](https://github.com/JasonLovesDoggo/RedditVideoMakerBot/commit/8b5670a8ebe17e8fa4f2bac00cb7ae2d40d5b290)

Is this an issue with my code, or the code as a whole?

**Comment by Proton45** [comment]
> > > What would happen if the program chooses the same thread a second time? Would it just overwrite the file or would it break? Either way, it would be very good if the program would just skip a thread that it has already been turned into a video. I thought of this, but I'm not sure how to implement it
> > 
> > 
> > already implemented in my fork
> > see [this commit](https://github.com/JasonLovesDoggo/RedditVideoMakerBot/commit/8b5670a8ebe17e8fa4f2bac00cb7ae2d40d5b290)
> 
> Is this an issue with my code, or the code as a whole?

Nope. It was just a question I had. Your code works great!

**Comment by Kamushy** [comment]
#167

**Comment by Kamushy** [comment]
Could somebody fix the conflicts, I'm at school currently and won't be able to fix it within 24 hours. @JasonLovesDoggo

**Comment by JasonLovesDoggo** [comment]
> What would happen if the program chooses the same thread a second time? Would it just overwrite the file or would it break? Either way, it would be very good if the programm would just skip a thread that it has already been turned into a video. I thought of this, but I'm not sure how to implement it

Sorry I'm at a friend's house and I won't be home until early tomorrow.

**Comment by Kamushy** [comment]
Resolved conflict

**Comment by jacesleeman** [comment]
should def be pushed into the master works well and so helpful

File: reddit/subreddit.py [hunk 1]
```diff
@@ -4,9 +4,8 @@
 from dotenv import load_dotenv
 import os
 
-
-def get_subreddit_threads():
-
+def get_askreddit_threads():
```
Comment (CallumIO) [inline reddit/subreddit.py:7]: Please change back to `get_subreddit_threads()` as that is the new method

File: video_creation/final_video.py [hunk 2]
```diff
@@ -7,14 +7,19 @@
     CompositeAudioClip,
     CompositeVideoClip,
 )
+import reddit.askreddit
```
Comment (CallumIO) [inline video_creation/final_video.py:10]: Should import `reddit.subreddit` to fit new file name

File: video_creation/final_video.py [hunk 3]
```diff
@@ -53,9 +58,7 @@ def make_final_video(number_of_clips):
     )
     image_concat.audio = audio_composite
     final = CompositeVideoClip([background_clip, image_concat])
-    final.write_videofile(
-        "assets/final_video.mp4", fps=30, audio_codec="aac", audio_bitrate="192k"
-    )
-
+    filename = (re.sub('[?\"%*:|<>]', '', ("assets/" + reddit.askreddit.submission.title + ".mp4")))
```
Comment (CallumIO) [inline video_creation/final_video.py:61]: Needs to be changed to new filename

**Review by CallumIO — CHANGES_REQUESTED** [review]
Looks good, just needs a few tweaks to fit with the new file structure and module names.

**Comment by Kamushy** [comment]
> Looks good, just needs a few tweaks to fit with the new file structure and module names.

Changes should be fixed now

**Review by CallumIO — APPROVED** [review]
LGTM!
Thanks for this 😄

**Review by jacesleeman — APPROVED** [review]

**Comment by CallumIO** [comment]
Thanks for this 😃

**Pull request merged**